magefile.go

```diff
@@ -3700,26 +3605,101 @@ func (Otel) GolangCrossBuild() error {
 	params := devtools.DefaultGolangCrossBuildArgs()
 	params.Name = "elastic-otel-collector-" + mage.Platform.GOOS + "-" + mage.Platform.Arch
 	params.OutputDir = "build/golang-crossbuild"
-	params.Package = "github.com/elastic/elastic-agent/internal/edot"
+	params.WorkDir = "internal/edot"
+	params.Package = "."
+	params.ExtraFlags = append(params.ExtraFlags, "-tags=agentbeat")
 	injectBuildVars(params.Vars)
 
+	// embedded packetbeat is only included in a non-FIPS build
+	if !mage.FIPSBuild {
+		// requires the NPCAP installer on Windows
+		if err := xpacketbeat.CopyNPCAPInstaller(filepath.Join("beats", "x-pack", "packetbeat", "npcap", "installer")); err != nil {
+			// to allow local builds for Windows, this is allowed to fail
```

Comment:
I think you want this to be an error so we can catch if this is happening in CI, the package target for windows/amd64 is built without -v so we can't even see if we hit these logs.